packages/contracts/src/vault/MaturityVault.sol

```diff
@@ -73,9 +78,8 @@ abstract contract MaturityVault is Vault {
     /**
      * @notice Enables/disables the Maturity Check according to the [status] value.
      * @dev 'Toggling' means flipping the existing state. This is simply a mutator.
-     * @param status Boolean value to toggle
      */
-    function _toggleMaturityCheck(bool status) internal {
-        checkMaturity = status;
```

Comment:
similar, let's keep the function taking a bool as an argument and rename it to be setMaturityCheck(bool _checkMaturity)